platforms/common/include/px4_platform_common/crypto.h

```diff
@@ -0,0 +1,219 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2021 Technology Innovation Institute. All rights reserved.
```

Comment:
Stupid question: shouldn't this be:

```suggestion
 *   Copyright (c) 2021 PX4 Development Team. All rights reserved.
```

Reply:
I am not really part of PX4 Development Team (?). This is developed in and for TII, and is a  contribution from TII to PX4. So I think the copyright is proper (there is lots of file in PX4 where the copyright is with someone outside of "PX4 Development Team" (e.g. other companies such as Sony, individual persons etc..)

I don't know if this matters in practice (the license dictates how the code can be used) though...

Reply:
Yer not too sure.

Reply:
You're absolutely entitled to do it like this if you like.

On a much more practical note it's ever so slightly easier from a tooling perspective to update the years automatically if the rest of the header is identical.